cmd/gossamer/main_test.go

```diff
@@ -228,6 +228,8 @@ func TestInvalidCommand(t *testing.T) {
 }
 
 func TestGossamerCommand(t *testing.T) {
+	t.Skip() // not sure how relevant this is anymore, it also slows down the tests a lot
```

Comment:
Should we add a TODO here, or a ticket to consider removing this?

Reply:
will do!